src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].toString();
```

Comment:
It's bad practice to initialize dependencies in the method. You can initialize it in fields, but good practice initialization using a constructor. So, please, move Random object initialization out of this method.

Reply:
Also, please use name() for getting String representation of enum constants, as described [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)